docs/GettingStarted.md

```diff
@@ -18,48 +32,56 @@ Open your app's `CMakeLists.txt`. This can be found under `External Build Files`
 Now add the following commands to the end of `CMakeLists.txt`. **Remember to update `**PATH TO OBOE**` with your local Oboe path from the previous step**:
 
     # Set the path to the Oboe directory.
-    set (OBOE_DIR ***PATH TO OBOE***) 
-    
+    set (OBOE_DIR ***PATH TO OBOE***)
+
     # Add the Oboe library as a subdirectory in your project.
-    add_subdirectory (${OBOE_DIR} ./oboe) 
-    
+    # add_subdirectory tells CMake to look in this directory to
+    # compile oboe source files using oboe's CMake file.
+    # ./oboe specifies where the compiled binaries will be stored
+    add_subdirectory (${OBOE_DIR} ./oboe)
+
     # Specify the path to the Oboe header files.
-    include_directories (${OBOE_DIR}/include)  
+    # This allows targets compiled with this CMake (application code)
+    # to see public Oboe headers, in order to access its API.
+    include_directories (${OBOE_DIR}/include)
 
 
-In the same file find the [`target_link_libraries`](https://cmake.org/cmake/help/latest/command/target_link_libraries.html) command. 
+In the same file find the [`target_link_libraries`](https://cmake.org/cmake/help/latest/command/target_link_libraries.html) command.
 Add `oboe` to the list of libraries which your app's library depends on. For example:
 
     target_link_libraries(native-lib oboe)
 
-Here's a complete example `CMakeLists.txt` file:
+Here's a complete example `CMakeLists.txt` file (without `OBOE_DIR` set):
```

Comment:
Why did you add "without OBOE_DIR set"? What are you trying to tell the reader?

Reply:
OBOE_DIR needs to be set for this build to compile. Right now it is simply a undefined variable.

Reply:
It's set a few lines below `set (OBOE_DIR ./oboe)`